nemo/collections/llm/api.py

```diff
@@ -409,6 +379,76 @@ def deploy(
     nm.stop()
 
 
+def evaluate(
+    nemo_checkpoint_path: Path,
+    url: str = "http://0.0.0.0:1234/v1",
```

Comment:
n00b question: why is a default set for this? the port doesn't match the default there either?